_download-meta/openttd-nightlies.md

```diff
@@ -1,10 +1,10 @@
 ---
-folder: nightlies/trunk
+folder_old_infrastructure: nightlies/trunk
 ---
 
 For OpenTTD you can use the original Transport Tycoon Deluxe data files (you need to own a Transport Tycoon Deluxe CD).
 There are also the free alternatives: [OpenGFX (graphics)](http://dev.openttdcoop.org/projects/opengfx), [OpenSFX (sound)](http://dev.openttdcoop.org/projects/opensfx), and [OpenMSX (music)](http://dev.openttdcoop.org/projects/openmsx).
 These can be installed automatically by the Windows and OS/2 installers.
-Please refer to the [readme](https://binaries.openttd.org/nightlies/trunk/@@version@@/readme.txt) for more information.
+Please refer to the [readme](https://openttd.ams3.cdn.digitaloceanspaces.com/openttd-nightlies/@@version@@/README.md) for more information.
```

Comment:
Given it's likely to change at some point, perhaps putting the URL in a variable somewhere might be appropriate?

Reply:
Yeah; more of these need attention, as too many things are hardcoded. But I have yet to find a way to make this easier to template (I now wrote a custom 'template engine', that replaces @@version@@). So I am going to create an issue out of this.